ait/core/server/server.py

```diff
@@ -286,7 +291,7 @@ def _create_plugin(self, config):
                                 zmq_args={'zmq_context': self.broker.context,
                                           'zmq_proxy_xsub_url': self.broker.XSUB_URL,
                                           'zmq_proxy_xpub_url': self.broker.XPUB_URL},
-                                **config
+                                **other_args
```

Comment:
Hi @seanlu99, this works, but I would like to propose a different approach. Instead of continuing to pass in the inputs, outputs, and other arguments separately, what if we change the initialization to just take in the entire `**config` as `kwargs`, like
```suggestion
        instance = plugin_class(zmq_args={'zmq_context': self.broker.context,
                                          'zmq_proxy_xsub_url': self.broker.XSUB_URL,
                                          'zmq_proxy_xpub_url': self.broker.XPUB_URL},
                                **config)
```
This would require editing the plugin parent class's initializer and moving the error checks for no inputs/outputs there.

Reply:
In addition, if we do this, for consistency I'd like to update the stream and handler classes to work the same way. This will also require updating of the relevant unit tests.